style.css

```diff
@@ -0,0 +1,421 @@
+html {
+  background-color: #e5e5e5;
+  font-family: 'Poppins', sans-serif;
+}
+
+/* Mobile version */
+
+@media only screen and (max-width: 768px) {
+  * {
+    margin: 0%;
+  }
+
+  header .nav-bar {
+    background-color: white;
+    padding: 30px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  header .my-logo {
+    font-weight: 700;
+    color: #6070ff;
+    text-decoration: none;
+    font-size: 20px;
+  }
+
+  header .nav-bar .button-b {
+    position: relative;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
```

Comment:
- [ ] Please Fix your style because when you choose a narrow mobile display mode the page content starts to overflow.
You can see this [article](https://www.w3schools.com/css/css3_box-sizing.asp) I think it will help you.